index.html

```diff
@@ -304,23 +305,9 @@
         </table>
       </div>   
           
-      <div id="id_sitefooter">
-        <table id="id_sitefooter_table0">
-          <script type="text/javascript" src="./scripts/static_text/sitefooter/content_sitefooter_table0.js">
+      <div id="id_sitefooter">	  
+        <script type="text/javascript" src="./scripts/static_text/sitefooter/content_sitefooter.js">
```

Comment:
I agree this cleans up the footer HTML a lot and gives more than enough capability (for the simple needs of the footer).